### ISSUE TITLE
Unable to connect with private key 

### ISSUE THREAD

**Issue by avdeev-a** [Eugeny/terminus#2018]
Mac OS HS 10.13.6
I am using private key for connection. 
I am adding a new host and trying to connect my remote machine:
_**Error while signing data with privateKey: error:06000066:public key routines:OPENSSL_internal:DECODE_ERROR**_
I can make the same exact connection with Iterm2 without any issues.

**Comment by ChipwizBen** [comment]
I have this issue with A100 also. According to [this issue](https://github.com/Eugeny/terminus/issues/666) the recommendation is to turn it into an OpenSSH key - however, it already is an OpenSSH key, albeit encrypted. The same key has worked with previous versions without a problem, and still works on regular Linux terminals.

**Comment by Eugeny** [comment]
Which cipher does your key use? (in the DEK-Info header in the file itself)

**Comment by ChipwizBen** [comment]
-----BEGIN RSA PRIVATE KEY-----
Proc-Type: 4,ENCRYPTED
DEK-Info: AES-128-CBC,A2241277DDF18FD1B4060B1D13B7D88F

**Comment by solomchuk** [comment]
Seeing the same issue with A102, using an unencrypted OpenSSH key. Works in all other terminals I've tried (Putty, Mobaxterm, built-in SSH in Win10, and various CentOS VMs).

**Comment by ChipwizBen** [comment]
Same for me, also on the newly released A102. I keep having to downgrade to A98 to get working again.

**Comment by ChipwizBen** [comment]
Same for me, also on the newly released A102. I keep having to downgrade to A98 to get working again. For what it's worth, I've removed the passphrase from the key and the issue persists. The new key still works with other applications including the regular Terminal.

The only thing that I can think of that might be different about my key and a "typical" key is that mine is a 4096 bit SHA256 RSA key. By default they're usually 2048 bit from ssh-keygen, though I notice that in the newest Fedora they're 3072 bit.

**Comment by solomchuk** [comment]
@ChipwizBen The key I'm using is 2048 bit RSA (default Puttygen settings), so I doubt that's the issue here.

**Comment by K3UL** [comment]
I also have an open issue on this, #2053 
It appeared in A99, with a new key parser.

**Comment by solomchuk** [comment]
@K3UL Yes, I originally used Puttygen to generate my key in ppk format. I then exported it to OpenSSH format and successfully used both in a number of settings, depending on what was supported. My understanding is that Terminus does not support ppk keys, so I tried using the OpenSSH version, which also does not work.

**Comment by ChipwizBen** [comment]
Sadly, this is not fixed in A103 either:

 X  Error while signing data with privateKey: error:06000066:public key routines:OPENSSL_internal:DECODE_ERROR

**Comment by tony-ruchti** [comment]
Seems to be something that was changed with release 99.
I installed back to release 98 (https://github.com/Eugeny/terminus/releases/tag/v1.0.98) and disabled updates, and can connect to my server through ssh with a private key generated from puttygen, updating to 99 causes an error of "failed to parse (unknown) as a valid auto format key" error

**Comment by tony-ruchti** [comment]
updated to 104, Here's the error I recieve when trying to connect with my puttygen key that was created:
Loading private key from  C:\Users\...ppk
 X  Failed to parse (unnamed) as a valid auto format key: key is not a private key

**Comment by iFlyinqMC** [comment]
Same issue ^

**Comment by Hassenky** [comment]
Got the same issue

**Comment by Eugeny** [comment]
Could you please test it against the today's build? Direct link: https://nightly.link/Eugeny/terminus/workflows/windows/master

**Comment by BearTS** [comment]
I can confirm, i have the same issue
It asks for the passphrase where it is an unencrypted ssh private key
